modules/admin/admin-ui/src/main/resources/web/admin/common/js/ui/selector/combobox/RichComboBox.ts

```diff
@@ -88,6 +88,15 @@ module api.ui.selector.combobox {
             }
         }
 
+        setReadOnly(readOnly: boolean) {
+            super.setReadOnly(readOnly);
+
+            this.comboBox.setReadOnly(readOnly);
+            this.selectedOptionsView.setEditable(!readOnly);
+
```

Comment:
Here we call .setEditable directly on this.selectedOptionsView, but that has already been done by the combobox itself on the previous line?